electronics/donts.md

```diff
@@ -63,9 +66,12 @@ Below is a non-exhaustive list of things that will get you into various kinds of
     * If you're reading this list, you probably can't do this safely.
 1. Using a multimeter in resistance mode on an energized circuit
     * Your measurements will be incorrect garbage, don't do it.
+1. Reading the result incorrectly.
+   * Pay attention to the SI prefix (especially m vs. M). It's easy to fall into the trap of assuming the prefix is what you expect and only looking at the number.
+   * When measuring resistance, results such as 0L and OVLD do not indicate 0 Ohms or a malfunction, but rather that the resistance is larger than the meter can measure, which, for normal meters, usually means it's basically an open. 

```

Comment:
"open" might be too jargony for a non EE, can we rephrase to be more normie friendly?